great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -17,6 +29,39 @@
 FULLY_QUALIFIED_PARAMETER_NAME_VALUE_KEY: str = "value"
 FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY: str = "details"
 
+attribute_name = Word(alphas, alphanums + "_.") + ZeroOrMore(
```

Comment:
Can you please add a comment that includes an example(s) of what you're trying parse here?

Reply:
@donaldheppner I added a docstring.  Please check.  P.S.: We should put on our TODO list adding similar docstrings to other usages of `pyparsing` (e.g., to support row conditions for metrics; evaluation parameters; and others).  Thanks!